src/app/constants/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const constants = {
```

Comment:
Please, check if we really need to wrap these constants into object?

Reply:
yes.Really need